### DISCUSSION TITLE
Getting metrics of video for  sensibility ,recall,f1 score,acurracy

### DISCUSSION THREAD

**Discussion by oal2727** [ultralytics/yolov5#10434] [Q&A]
Hi guys, I'm new to using yolo, I've been here for a couple of weeks now, and well, my question is regarding obtaining metrics for the detection of each video, well, the procedure is the usual one. I already have my model trained "file.pt" and I also have I have exported it, to load it in different computers, well in this case I have loaded it in my google colab for the detection of a video, and well what I need is to extract the same metrics if possible from what is displayed on the tensorboard I have understands that the tensorboard reference comes out when executing the train.py

**Comment by glenn-jocher** [comment]
@oal2727 for online logging dashboard see Comet or ClearML in our README:


<img width="873" alt="Screenshot 2022-12-08 at 15 39 37" src="https://user-images.githubusercontent.com/26833433/206589648-6e327050-db28-4a80-a78b-55f61f985cf7.png">
